src/manager.ts

```diff
@@ -184,16 +184,24 @@ export class Manager extends Disposable {
 
 				if (!workspace) {
 					// no workspace; try to decode endpoint to fix file
-					file = this._endpointManager.decodeLooseFileEndpoint(file);
-					if (!file) {
+					const potentialFile = this._endpointManager.decodeLooseFileEndpoint(file);
```

Comment:
nit: seems like you could just reassign `file` here and skip adding the `potentialFile` variable

Reply:
Oh, I just have it like that because I wanted to keep `file` strictly of type string. `potentialFile` is of type string | undefined and it gets checked for undefined before being assigned to `file`